addon/templates/components/cfb-form-editor/question/options.hbs

```diff
@@ -8,7 +8,7 @@
       model=row
     as |f|}}
       <div uk-grid class="uk-grid-small uk-flex">
-        {{#if (gt optionRows.length 1)}}
+        {{#if row.isNew}}
```

Comment:
I think this should be `(and row.isNew (gt optionRows.length 1))` otherwise it would be possible to remove the first and only row..

Reply:
Good point!